base/task.jl

```diff
@@ -1191,35 +1191,44 @@ end
 
 checktaskempty = Partr.multiq_check_empty
 
-@noinline function poptask(W::StickyWorkqueue)
+function wait()
+    ct = current_task()
+    # [task] user_time -yield-or-done-> wait_time
+    record_running_time!(ct)
+    # let GC run
+    GC.safepoint()
+    # check for libuv events
+    process_events()
+
+    # get the next task to run
+    result = nothing
+    have_result = false
+    W = workqueue_for(Threads.threadid())
     task = trypoptask(W)
     if !(task isa Task)
+        # didn't find a task to run, try again
         process_events()
         task = trypoptask(W)
     end
```

Comment:
this seems unneeded now (not bad, just unnecessary)

Reply:
Will remove if I "feel" no added latency at the REPL.

Reply:
Removed.